features/manage-position/Create.tsx

```diff
@@ -10,7 +10,7 @@ import EmpState from "../../containers/EmpState";
 import Totals from "../../containers/Totals";
 import Position from "../../containers/Position";
 
-import { useEtherscanUrl } from "./useEtherscanUrl";
+import { useEtherscanUrl } from "../../utils/useEtherscanUrl";
```

Comment:
I wonder why this diff is showing up I thought it should be merged into `develop` now have you merged with develop since I merged in the WETH PR?

Reply:
I made the same changes you did. I'll rebase now and it should remove it.